package.json

```diff
@@ -36,7 +36,7 @@
     "LICENSE",
     "README.md"
   ],
-  "homepage": "https://github.com/twilio-labs/plugin-serverless",
+  "homepage": "https://github.com/kaiquelupo/plugin-twilio-infra",
```

Comment:
Maybe using [$ref](https://swagger.io/docs/specification/using-ref/) is overkill for this JSON file, but let's remember to update this URL later.

Reply:
Very good point @bermudezmt!